src/Sql/dbo/Functions/UserCipherDetails.sql

```diff
@@ -17,12 +17,19 @@ SELECT
     CASE
         WHEN
             OU.[AccessAll] = 1
-            OR CU.[ReadOnly] = 0
             OR G.[AccessAll] = 1
-            OR CG.[ReadOnly] = 0
+            OR COALESCE(CU.[ReadOnly], CG.[ReadOnly], 0) = 0
         THEN 1
         ELSE 0
     END [Edit],
+    CASE
+        WHEN
+            OU.[AccessAll] = 1
+            OR G.[AccessAll] = 1
+            OR COALESCE(CU.[HidePasswords], CG.[HidePasswords], 0) = 0
```

Comment:
I changed the `HidePassword` / `ViewPassword` logic to be explicit about `AccessAll` as well, to be consistent since they already had that behavior.